pkgs/os-specific/linux/hostapd/default.nix

```diff
@@ -13,7 +13,7 @@ stdenv.mkDerivation rec {
   buildInputs = [ libnl openssl sqlite ];
 
   patches = [
-    (fetchurl {
+    (fetchpatch {
```

Comment:
switching that forces a hash change.

Reply:
It is guaranteed, that it does.

Reply:
https://github.com/NixOS/nixpkgs/pull/119347 <= we merged that 1h ago, the hash is going to change anyways.